recipes/pugixml/all/conanfile.py

```diff
@@ -0,0 +1,107 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class PugiXmlConan(ConanFile):
+    name = "pugixml"
+    version = "1.10"
```

Comment:
```suggestion
```

It should be generic, according the folder name.

Reply:
Thank you, I've missed this point.